i18n/locales/en.ts

```diff
@@ -3022,7 +3025,9 @@ Stack Hint: To spend before the secret is revealed, Vanderpoole uses his signatu
       output_zero_sig: 'Output 0 Signature',
       output_one_sig: 'Output 1 Signature',
       sats_distribution:
-        'Remember that Laszlo should only be receiving enough sats for the beer. Also we need to set aside some sats for the fees incase our channel closes!',
```

Comment:
I removed the part about setting fees aside for channel closes before I realized this same hint is being displayed on all the chapter 10 resource pages. It doesn't belong in the first few exercises I've done so far, but I can see the need to bring it back for a resource page in a later exercise.

Most of the resource pages need to be filled in but I want to get through chapter 10 QA before taking a closer look.

Reply:
yeah i am thinking we should be consistent with fees for all txs now, which means we start with the "deposit" 101,000 sats, pay a fee on the funding tx so our open channel has 100,000 and then all commitment txs pay another fee, meaning the Laszlo + You balance should = 99,000 for the rest of the chapter

Reply:
Good point! I linked your comment in this related issue that I filed: https://github.com/saving-satoshi/saving-satoshi/issues/1175

Reply:
You can decide where to delete these fee comments individually here.
https://github.com/saving-satoshi/saving-satoshi/blob/dev/content/resources/chapter-10/making-a-payment-2.tsx#L19

Reply:
that is just one resource page but it is in all of them

Reply:
Its only incorrect after we buy a second beer, then it could just read "Laszlo should receive enough sats for the beer(s)" ;-) It is true he doesnt pay miner fees, but we could also mention that the miner fee for the commitment comes out of your change (get ready for advanced Saving Satoshi: anchor outputs and package relay!)

Reply:
Thanks all! Here are the updates I made to the help pages:

1. Kept the help page for the first exercise (opening-a-channel-2) empty because everything is in the instructions.
2. Updated the help page for the multisig excercise (opening-a-channel-4) to say `Don't forget to set aside 1,000 sats for miner fees.`
3. Updated all refund transaction exercises (updating-the-state-1, updating-the-state-3, updating-the-state-5) so the help page says: `Don't forget to set aside 1,000 sats from your change for miner fees.` I did this because it doesn't make sense to mention Laszlo and the beer(s) in those, and to clarify that the miner fees come from your change.
4. Updated the help page for the exercises where you actually buy the beer (making-a-payment-2, making-a-payment-5) to say `Laszlo should receive enough sats for the beer, and is not responsible for the 1,000 sat miner fees. The fee comes out of your change.`, which is more or less what this PR initially intended.
5. Updated the help page for the last exercise where you buy the second beer to just say `Don't forget to set aside 1,000 sats from your change for miner fees.`. Hopefully at this point the user gets it :)